lib/Templating/Renderer/IncludeRenderer.php

```diff
@@ -89,7 +89,7 @@ public function render($include, array $params = [], $editmode = false, $cacheEn
         $cacheKey = null;
         $cacheConfig = false;
 
-        if ($cacheEnabled) {
+        if ($cacheEnabled && !$editmode) {
```

Comment:
There seems to be something wrong - this changes come from a commit which doesn't belong to this PR. 
@BlackbitNeueMedien can you please check? 
Thanks!

Reply:
Sorry, wrong base branch. Now it is correct, @brusch

Reply:
👍 That looks better, thanks a lot!